# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1349245418 14400
#      Wed Oct 03 02:23:38 2012 -0400
# Node ID 1865feebb41572e699c98debae28b8568842512f
# Parent  457a2ff2a71b0bed560c41bf29c96a02c786c238
* octave.cc (info_program, info_file): Delete unused static variables.

diff --git a/libinterp/octave.cc b/libinterp/octave.cc
--- a/libinterp/octave.cc
+++ b/libinterp/octave.cc
@@ -161,24 +161,16 @@ static std::string doc_cache_file;
 // The value for "EXEC_PATH" specified on the command line.
 // (--exec-path)
 static std::string exec_path;
 
 // The value for "IMAGE_PATH" specified on the command line.
 // (--image-path)
 static std::string image_path;
 
-// The value for "info_file" specified on the command line.
-// (--info-file)
-static std::string info_file;
-
-// The value for "info_program" specified on the command line.
-// (--info-program)
-static std::string info_program;
-
 // If TRUE, ignore the window system even if it is available.
 // (--no-window-system)
 static bool no_window_system = false;
 
 // The value for "texi_macros_file" specified on the command line.
 // (--texi-macros-file)
 static std::string texi_macros_file;
 
